server/Dockerfile

```diff
@@ -153,6 +154,9 @@ ARG PHP_VER
 # Make a copy of the configurations, so we can sync from it
     RUN cp -R /var/www/MISP/app/Config /var/www/MISP/app/Config.dist
 
+    # Intall Open ID Connect dependency
+    RUN cd /var/www/MISP/app && php composer.phar require jumbojett/openid-connect-php
```

Comment:
This should probably be done differently. But how?

Reply:
Maybe append the file here: https://github.com/coolacid/docker-misp/blob/9977d9281ffe5e843cd3a0ee73601c9158c2dbe3/server/Dockerfile#L4

Reply:
Done

Reply:
I wonder if a PR should be made to MISP to include this requirement in the composer.json file?

Reply:
Its listed as suggested dependency already.

https://github.com/MISP/MISP/blob/2.4/app/composer.json

Reply:
Ah, Ok I haven't had to work with composer in many many years. This looks like the right way to do this.